### DISCUSSION TITLE
Safe information separately in the same poll

### DISCUSSION THREAD

**Discussion by OptiNex** [rubenarslan/formr.org#533] [Q&A]
Dear all, 

for our master's thesis, my friend and I would like to know how to safe mail adresses separately from the main data set but in the same poll. 
Is there a way to do so with formR? Also, is there a way to safe a personal code and the email adress in the same poll but with another data set? We would like to have one data set with the answers to the items and the personal code and one data set with the personal code and the email adresses, but in the same poll. 

Thanks in advance!
Marlene

**Comment by rubenarslan** [comment]
Yes, this is possible. Create two separate surveys. For the survey with identifying information, go to survey settings and check one or both of these checkboxes.
<img width="973" alt="image" src="https://github.com/rubenarslan/formr.org/assets/831109/2d59a2aa-1b58-4d82-8b54-dd60377d11f7">

**Reply by OptiNex** [reply]
Okay, thank you very much! Is it possible to connect the two surveys so that people can answer the surveys within one run?

**Reply by rubenarslan** [reply]
Yes, that's the idea. You put both surveys in the same run. formr can connect the information, so somebody can, for example, be invited via an email based on information in a separate survey. But for you, the researcher, the email addresses are not linked to the research data.

**Reply by OptiNex** [reply]
Dear Ruben, 
thanks so much for your quick response, it worked out perfectely!

**Comment by OptiNex** [comment]
Hi Ruben, 
sorry to bother you again.
We still have one question: For our survey we importet our items with an excel spreadsheet. Unfortunaley, the output of the multiple choice answer ranges from 1 to X, but we're using one standardized questionnaire where the answers need to range from 0 to x for a correct analysis. Is it possible to change the numeric output? Also, this only concerns a few items within the same survey, but the other items should have a format (1 to x), so is there a way, that the data saved for the multiple choice answers differ within the same survey? 
Thanks in advance!

**Comment by rubenarslan** [comment]
Yes, this is possible. You can define the values stored for multiple choices in a second choices sheet, see the documentation and example here: https://formr.org/documentation#sample_choices_sheet.
